cmd/earthly/debug_cmds.go

```diff
@@ -104,7 +104,6 @@ func (app *earthlyApp) actionDebugBuildkitInfo(cliCtx *cli.Context) error {
 	fmt.Printf("Buildkit revision: %s\n", info.BuildkitVersion.Revision)
 	fmt.Printf("Buildkit package: %s\n", info.BuildkitVersion.Package)
 	fmt.Printf("Num sessions: %d\n", info.NumSessions)
-	fmt.Printf("Seconds idle: %d\n", info.SecondsIdle)
```

Comment:
Intended change?

Reply:
This one was intentional actually because the seconds Idle always shows zero now.  I.e. we reset to zero through the Reserve call as we're connecting to it in order to print the debug info. 

We could change the code so that it doesn't always call the reserve endpoint, but I didn't think that would be worth it.

Reply:
Oh yeah, true